File: MultiBoxClientJavaCore/src/sk/hackcraft/multibox/net/data/GetPlaylistResultData.java
package sk.hackcraft.multibox.net.data;

import java.util.Collections;
import java.util.LinkedList;
import java.util.List;

import sk.hackcraft.multibox.model.Multimedia;

public class GetPlaylistResultData
{
	private final List<Multimedia> playlist;
	
	public GetPlaylistResultData(List<Multimedia> playlist)
	{
		this.playlist = Collections.unmodifiableList(new LinkedList<Multimedia>(playlist));
	}
	
	public List<Multimedia> getPlaylist()
	{
		return playlist;
	}
}

File: MultiboxClientCSharpCore/Net/Data/GetPlaylistResultData.cs
using MultiboxClientCSharpCore.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace MultiboxClientCSharpCore.Net.Data
{
	public class GetPlaylistResultData
	{
		public GetPlaylistResultData(IList<MultimediaItem> playlist)
		{
			this.Playlist = new List<MultimediaItem>(playlist);
		}

		public IList<MultimediaItem> Playlist
		{
			get;
			private set;
		}
	}
}
